recipes-graphics/imx-gpu-apitrace/imx-gpu-apitrace/0004-cli-Pickle-blobs-as-Python-3-bytes-objects.patch

```diff
@@ -0,0 +1,82 @@
+From 63b651332cfeb02cc723675ad2d150641d037e21 Mon Sep 17 00:00:00 2001
+From: Jose Fonseca <jfonseca@vmware.com>
+Date: Sun, 28 Apr 2019 13:47:29 +0100
+Subject: [PATCH 4/7] cli: Pickle blobs as Python 3 bytes objects.
+
+Upstream-Status: Pending
```

Comment:
Here too

Reply:
I've checked the latest state of `imx_7.1` branch and looks like this patch applies. It modifies generated blobs from using _bytearrays_ to python3 _byte objects_.

Original support for _bytearrays_ has been introduced with eba2deca7 from upstream, and is not changed at the HEAD of `imx_7.1`.

Reply:
I thought about this a bit, and actually this patch is not really crucial here to have python3 support. Originally, it has been introduced on the `python3` branch in upstream together with all other modifications required to move to python3, that's what made me think this one is also necessary.

I've tried to exclude this patch from recipe and build succeeded.

@otavio @thochstein Can you please advise whether you'd prefer this patch should be included in the series or should I drop it?

Thanks a lot!

-- andrey

Reply:
I've dropped this patch as well and updated PR with new version.